scripts/emulator_read_target.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# this script needs to be run with export to work.
+
+output_dir="android/app/build/outputs/emulator_output"
```

Comment:
if we make this an absolute path with `git rev-parse --show-toplevel` then everything that uses `output_dir` is protected from running in wrong directory... however be careful to remove the `./` from paths in the code with `./${output_dir}/foo/bar`